backend/src/main/java/com/htecgroup/skynest/lambda/LambdaType.java

```diff
@@ -1,5 +1,16 @@
 package com.htecgroup.skynest.lambda;
 
 public enum LambdaType {
-  UPLOAD_FILE_TO_EXTERNAL_SERVICE_LAMBDA
+  // The code is used to represent the lambdas in the database and as such should not be changed.
+  UPLOAD_FILE_TO_EXTERNAL_SERVICE_LAMBDA("U_F_T_E_S_L");
+
+  private String code;
```

Comment:
or text, rather than code?
this makes it look like it's the lambda's source code or something

Reply:
maybe databaseCode, since that's the value under which we are storing the enum.

Reply:
sure, or databaseString or something

or you could redefine toString for the enum, though that might cause problems later